themes/jb/layouts/partials/people/horizontal.html

```diff
@@ -1,6 +1,6 @@
-<div class="card is-horizontal">
-  <div class="columns">
-    <div class="column is-one-third pr-0">
+<div class="card is-horizontal pt-4">
+  <div class="columns section">
+    <div class="column is-one-third pr-0 px-0 pt-0 pb-1 columns mb-0">
```

Comment:
`px-0` should cover what `pr-0` is doing
```suggestion
    <div class="column is-one-third px-0 pt-0 pb-1 columns mb-0">
```